expression/builtin_info.go

```diff
@@ -346,6 +347,39 @@ func (b *builtinTiDBVersionSig) evalString(_ types.Row) (string, bool, error) {
 	return printer.GetTiDBInfo(), false, nil
 }
 
+type tidbIsDDLOwnerFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *tidbIsDDLOwnerFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
+	if err := c.verifyArgs(args); err != nil {
+		return nil, errors.Trace(err)
+	}
+	bf := newBaseBuiltinFuncWithTp(ctx, args, types.ETInt)
+	sig := &builtinTiDBIsDDLOwnerSig{bf}
+	return sig, nil
+}
+
+type builtinTiDBIsDDLOwnerSig struct {
+	baseBuiltinFunc
+}
+
+func (b *builtinTiDBIsDDLOwnerSig) Clone() builtinFunc {
+	newSig := &builtinTiDBIsDDLOwnerSig{}
+	newSig.cloneFrom(&b.baseBuiltinFunc)
+	return newSig
+}
+
+// evalInt evals a builtinTiDBIsDDLOwnerSig.
+func (b *builtinTiDBIsDDLOwnerSig) evalInt(_ types.Row) (res int64, isNull bool, err error) {
+	ddlOwnerChecker := b.ctx.DDLOwnerChecker()
+	if ddlOwnerChecker.IsOwner() {
+		res = 1
```

Comment:
Could we replace `1` with `true`? I think it's more readable.

Reply:
1 is enough

Reply:
You can use `1`, but need to write the documentation